portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Bernardo's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Bernardo's Portfolio</h1>
+      <p>This is Bernardo's portfolio.</p>
+      <p>A message from Bernardo:</p>
+      <p>
+          "Hi, everyone! It is great to have you here! My name is Bernardo Eilert Trevisan and I am currently 20 years old. 
+          I was born in a city located in the south of Brazil called Porto Alegre. I lived there with my parents and my brother until I graduated from High School.
+          After that, I moved to Connecticut to begin my undergraduate studies at Yale University. As of today, I am a rising junior majoring in 
+          Computer Science. I am extremely excited to be working as a STEP intern for Google this Summer and cannot wait to see this portfolio grow 
+          as I make my way through the upcoming projects."
+      </p>
+      <img src="images/bernardo.jpg"/>
```

Comment:
Maybe add an ID to this img and apply CSS styling to that (instead of the img tag). Since you have positioned this image a certain way in style.css, if you decide to add more images later, that styling will be applied to all img tags.

Reply:
That does make a lot of sense. Thanks!